tests/gordo/builder/test_builder.py

```diff
@@ -642,3 +643,38 @@ def test_setting_seed(seed, model_config):
         assert df1.equals(df2)
     else:
         assert not df1.equals(df2)
+
+
+@pytest.mark.parametrize(
+    "cv",
+    (
+        {"sklearn.model_selection.TimeSeriesSplit": {"n_splits": 5}},
+        {
+            "sklearn.model_selection.TimeSeriesSplit": {
+                "n_splits": 5,
+                "max_train_size": 10,
+            }
+        },
+        {"sklearn.model_selection.ShuffleSplit": {"n_splits": 5}},
+    ),
+)
+@patch("gordo.serializer.pipeline_from_definition")
+def test_n_splits_from_config(MockClient, cv):
```

Comment:
this is nit picking, I admit, but maybe rename this to `mocked_pipeline_from_definition` ?